dataflux_pytorch/benchmark/checkpointing/multinode/fsspecfsdp.py

```diff
@@ -0,0 +1,109 @@
+import os
```

Comment:
Can we move this entire file to demo and an example in demo and use that demo as base in benchmarks train.py 

I think this would make it consistent with multi node benchmarks.

Reply:
Done.